docker/kayobe/Dockerfile

```diff
@@ -7,6 +7,11 @@ ARG BASE_IMAGE="quay.io/centos/centos:stream8"
 FROM ${BASE_IMAGE}
 MAINTAINER "Will Szumski" <will@stackhpc.com>
 
+ARG http_proxy
+ARG https_proxy
+ENV http_proxy $http_proxy
+ENV https_proxy $https_proxy
```

Comment:
It's quite common to also need no_proxy, but I note that Will had [problems will commas](https://github.com/stackhpc/stackhpc-kayobe-config/blob/402ce66c4b3820c3949243d70f45d2aa4af21def/.github/workflows/stackhpc-build-kayobe-image.yml#L69).